python/example_code/route53-recovery-cluster/update_routing_control_states.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Replace this copyright with our standard copyright notice:

\#   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
\#   SPDX-License-Identifier: Apache-2.0

Reply:
Updated copyright for the python scripts